### PR TITLE
fix: remove the statement of session number limits

### LINKED ISSUE #1927: Spanner: Doc Bug Cloud Spanner no longer has the 10K "Sessions per database per node" limit.
https://cloud.google.com/java/docs/reference/google-cloud-spanner/6.21.2/com.google.cloud.spanner.Session

It says, "Cloud Spanner limits the number of sessions that can exist at any given time"

### PR DESCRIPTION
**Pull request by rbrtwng**
Cloud Spanner no longer has the 10K "Sessions per database per node" limit.

Doc fix only

Fixes #1927

### CODE REVIEW

**Review by ansh0l — APPROVED** [review]
LGTM

**Comment by ansh0l** [comment]
(will wait for presubmits to run, since this is a doc string fix, adding automerge label)

**Comment by ansh0l** [comment]
@rbrtwng : It looks like a couple of (required) presubmits fail because of minor nits.

1. Linting  line 34 seems to have an additional whitespace at end
2. Conventional commits: since commit message and PR subject have different title. 

Please add them and tag me, I will run the presubmits again after them.

**Comment by rbrtwng** [comment]
@ansh0l can you please run presubmit again?

File: google-cloud-spanner/src/main/java/com/google/cloud/spanner/Session.java [hunk 1]
```diff
@@ -28,10 +28,10 @@
...
  * queries use a transaction internally, and count toward the one transaction limit.
  *
- * <p>Cloud Spanner limits the number of sessions that can exist at any given time; thus, it is a
- * good idea to delete idle and/or unneeded sessions. Aside from explicit deletes, Cloud Spanner can
- * delete sessions for which no operations are sent for more than an hour, or due to internal
- * errors. If a session is deleted, requests to it return {@link ErrorCode#NOT_FOUND}.
+ * <p>It is a good idea to delete idle and/or unneeded sessions. Aside from explicit deletes,
+ * Cloud Spanner can delete sessions for which no operations are sent for more than an hour,
+ * or due to internal errors. If a session is deleted, requests to it return
+ * {@link ErrorCode#NOT_FOUND}.
```
Comment (ansh0l) [inline google-cloud-spanner/src/main/java/com/google/cloud/spanner/Session.java:34]: @rbrtwng : Please remove the whitespace at the end of this line, its failing one of the required presubmits for linting.

**Review by ansh0l — APPROVED** [review]
LGTM, but need to make a minor change to satisfy the lint presubmit.

**Pull request merged**